src/Agent/Inqueue.vue

```diff
@@ -56,7 +56,8 @@ export default {
         state.inqueue ? this.inqueue_record = state.inqueue.record : this.inqueue_record = null       
         if (state.inqueue && state.inqueue.record == 'inqueue_call') {
           this.call_visible = true
-          this.uuid = state.inqueue.uuid
+          if (state.inqueue.uuid !== undefined)
```

Comment:
if (state.inqueue.uuid) ?

Reply:
Its not good solution at all... I just look on CDR reports and disposition is not there even it was selected in wrap-up mode.(i see it when i select disposition in oncall state) The reason why disposition was not visible is that state.inqueue.uuid is undefined in wrap-up mode.